Sources/Basic/Path.swift

```diff
@@ -485,6 +499,16 @@ extension AbsolutePath {
     public func contains(_ other: AbsolutePath) -> Bool {
         return self.components.starts(with: other.components)
     }
+
+    /// Checks the path and throws a `PathError` if path is invalid.
+    ///
+    /// This method is strictly syntactic and does not access the file system
```

Comment:
Our path APIs don't access filesystem anyway so I don't think this comment is needed.